HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -3695,7 +3710,7 @@ def from_oga(hpxml)
   end
 
   class Refrigerator < BaseElement
-    ATTRS = [:id, :location, :rated_annual_kwh, :adjusted_annual_kwh, :usage_multiplier]
+    ATTRS = [:id, :location, :rated_annual_kwh, :adjusted_annual_kwh, :usage_multiplier, :primary_indicator]
```

Comment:
This needs to be called out in EPvalidator. Is it optional or required?

Reply:
Optional. I added `'[not(PrimaryIndicator)] | PrimaryIndicator[text()="true" or text()="false"]' => one` to [Refrigerator].

Reply:
You'll want to default the primary indicator in measure.rb if it's not provided then. I assume the logic is to set it to true for the first HPXML fridge if there are no fridges with it set to true, and false for every other case?

And we probably want error-checking in hpxml.rb if more than one fridge have it set to true.

Reply:
On the flip side, what if all but one refrigerators are set to false. Then you can assume the one without primary indicator set is the primary refrigerator?

Seems easier to require setting primary indicator.

Reply:
I'm okay with requiring it. 

I also realized my error-checking proposal wasn't general enough. We should check that, if there are refrigerators in the HPXML file, exactly one is set as primary.